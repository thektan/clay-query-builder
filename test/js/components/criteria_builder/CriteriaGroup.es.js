import React from 'react';
import CriteriaGroup from 'components/criteria_builder/CriteriaGroup.es';
import {cleanup, render} from 'react-testing-library';

const connectDnd = jest.fn(el => el);

describe(
	'CriteriaGroup',
	() => {
		afterEach(cleanup);

		it(
			'should render',
			() => {
				const OriginalCriteriaGroup = CriteriaGroup.DecoratedComponent;

				const {asFragment} = render(
					<OriginalCriteriaGroup
						connectDragPreview={connectDnd}
					/>
				);

				expect(asFragment()).toMatchSnapshot();
			}
		);
	}
);