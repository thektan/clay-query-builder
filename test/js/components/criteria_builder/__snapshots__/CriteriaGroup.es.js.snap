// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CriteriaGroup should render 1`] = `
<DocumentFragment>
  <div
    class="criteria-group-root criteria-group-item"
  />
</DocumentFragment>
`;
